app/models/stop.rb

```diff
@@ -71,23 +77,20 @@ def csv_row_values
       :not_served_by,
       :identified_by,
       :not_identified_by,
+      :parent_stop_onestop_id,
+      :includes_stop_transfers,
+      :does_not_include_stop_transfers
     ],
     protected_attributes: [
       :identifiers,
       :last_conflated_at
     ]
   })
   def after_create_making_history(changeset)
-    OperatorRouteStopRelationship.manage_multiple(
-      stop: {
-        served_by: self.served_by || [],
-        not_served_by: self.not_served_by || [],
-        model: self
-      },
-      changeset: changeset
-    )
+    before_update_making_history(changeset)
   end
   def before_update_making_history(changeset)
+    super(changeset)
     OperatorRouteStopRelationship.manage_multiple(
```

Comment:
Lots going on here. How about separating 103-106 and 107-133 into two separate private methods with descriptive names? Then you can call them in sequence from both `after_create_making_history` and `before_update_making_history`.